tools/gnur-runtime/src/main/java/org/renjin/gnur/api/CCallablesRegister.java

```diff
@@ -0,0 +1,18 @@
+package org.renjin.gnur.api;
+
+import java.lang.invoke.MethodHandle;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CCallablesRegister {
+  static final Map<String,MethodHandle> callMap = new HashMap<>();
```

Comment:
You only need this lazy-initialization pattern if the initialization code is very expensive in terms of runtime or memory. Here that is not the case. move `callMap` to a `private static final` member of `Rdynload` to simplify. Also observe java naming conventions: should be named `CALL_MAP`.